templates/registration/confirmation_email.txt.twig

```diff
@@ -0,0 +1,10 @@
+Hi,
+
+Please confirm your email address by clicking the following link:
+
+{{ signedUrl|raw }}
```

Comment:
`|raw` is useless. A `.txt.twig` file already disables autoescaping everywhere (plain text does not require escaping)